scripts/tags_and_languages.py

```diff
@@ -408,6 +426,9 @@ def run_modifications(conn_db, tag_type, mapping_languages_countries, post_call_
         print(f"prepare log table {tag_type} - time {datetime.now()-time_start}")
         create_and_populate_table(conn_db, tag_type, tags_to_update)
 
-    print(f"update products {tag_type} - time {datetime.now()-time_start}")
-    run_modifications(conn_db, tag_type, mapping_languages_countries, post_call_url, headers)
-
+    if args.modify:
+        print(f"update products {tag_type} - time {datetime.now()-time_start}")
+        run_modifications(conn_db, tag_type, mapping_languages_countries, post_call_url, headers)
+    else:
+        all_rows = conn_db.execute(f"SELECT * FROM products_to_update_{tag_type} WHERE updated = FALSE").fetchall()
+        print(f"There are {len(all_rows)} products to be updated for {tag_type}")
```

Comment:
Maybe add a line on how one can inspect the programmed modifications using duckdb ?

Reply:
ok, done.